src/main/java/com/ericsson/ei/mongo/MongoUri.java

```diff
@@ -0,0 +1,116 @@
+/*
+   Copyright 2019 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.mongo;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class MongoUri {
+
+    private static final String HIDDEN_PASSWORD = "********";
+
+    /**
+     * This method replaces the given password if any.
```

Comment:
Maybe you could be more clear in this description. Something like:

"This method hides the password in a given string and returns it.

Reply:
Done.